recipes/py_mini_racer/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "py_mini_racer" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/py_mini_racer-{{ version }}.tar.gz
+  sha256: f71e36b643d947ba698c57cd9bd2232c83ca997b0802fc2f7f79582377040c11
+
+build:
+  skip: True  # [linux]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - py_mini_racer
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/sqreen/PyMiniRacer
+  summary: Minimal, modern embedded V8 for Python.
+  license: MIT
```

Looks like it is not MIT. See https://github.com/sqreen/PyMiniRacer/blob/v0.6.0/setup.py#L47

```suggestion
  license: ISCL
```